bloopark-addons/bp_food/views/food_customer_view.xml

```diff
@@ -6,96 +6,46 @@
     <!-- Customer Data Master -->
     <!-- Form view of new customer data -->
     <record id="view_bp_food_customers_form" model="ir.ui.view">
-        <field name="name">bp_food_customers.form</field>
-        <field name="model">bp.food.customers</field>
+        <field name="name">res.partner.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
         <field name="arch" type="xml">
-            <form string="Customer Detail">
-                <sheet>
-                    <group>
-                        <field name="name" colspan="1"/>
-                        <newline/>
-                        <field name="number"/>
-                        <newline/>
-                        <field name="email"/>
-                    </group>
-                </sheet>
-            </form>
+            <field name="vat" position="after">
+                <field name="orientation" select="0"/>
+            </field>
+            <field name="vat" position="after">
+                <field name="frequency" select="0"/>
+            </field>
         </field>
     </record>
 
-    <!-- Tree view of hotel restaurant tables -->
+    <!--     Tree view of hotel restaurant tables -->
     <record id="view_bp_food_customers_tree" model="ir.ui.view">
-        <field name="name">bp_food_customers.tree</field>
-        <field name="model">bp.food.customers</field>
+        <field name="name">res.partner.tree</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_tree"/>
         <field name="arch" type="xml">
-            <tree string="Customer Details">
-                <field name="name"/>
-                <field name="number"/>
-                <field name="email"/>
-            </tree>
+            <field name="vat" position="after">
+                <field name="orientation" select="0"/>
```

Comment:
Why you are using select=0 here? If what you want is having a default value selected you should do it in the model, in the field definition using default='XXXXX'